[2025-02-09T15:09:45.563+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-09T15:09:45.593+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_postgres_and_csv_data.wait_for_extract_postgres_and_csv_data manual__2025-02-09T15:09:40.814819+00:00 [queued]>
[2025-02-09T15:09:45.607+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_postgres_and_csv_data.wait_for_extract_postgres_and_csv_data manual__2025-02-09T15:09:40.814819+00:00 [queued]>
[2025-02-09T15:09:45.608+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-09T15:09:45.635+0000] {taskinstance.py:2889} INFO - Executing <Task(ExternalTaskSensor): wait_for_extract_postgres_and_csv_data> on 2025-02-09 15:09:40.814819+00:00
[2025-02-09T15:09:45.650+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1936) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-09T15:09:45.654+0000] {standard_task_runner.py:72} INFO - Started process 1946 to run task
[2025-02-09T15:09:45.653+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_postgres_and_csv_data', 'wait_for_extract_postgres_and_csv_data', 'manual__2025-02-09T15:09:40.814819+00:00', '--job-id', '126', '--raw', '--subdir', 'DAGS_FOLDER/Load_CSV_to_Postgres.py', '--cfg-path', '/tmp/tmpbcin8fr0']
[2025-02-09T15:09:45.656+0000] {standard_task_runner.py:105} INFO - Job 126: Subtask wait_for_extract_postgres_and_csv_data
[2025-02-09T15:09:45.745+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_postgres_and_csv_data.wait_for_extract_postgres_and_csv_data manual__2025-02-09T15:09:40.814819+00:00 [running]> on host 8c410daec8d3
[2025-02-09T15:09:45.910+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_postgres_and_csv_data' AIRFLOW_CTX_TASK_ID='wait_for_extract_postgres_and_csv_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-09T15:09:40.814819+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-09T15:09:40.814819+00:00'
[2025-02-09T15:09:45.913+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-09T15:09:45.914+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-09T15:09:45.915+0000] {logging_mixin.py:190} INFO - Current task name:wait_for_extract_postgres_and_csv_data state:running start_date:2025-02-09 15:09:45.594863+00:00
[2025-02-09T15:09:45.915+0000] {logging_mixin.py:190} INFO - Dag name:extract_postgres_and_csv_data and current dag run status:running
[2025-02-09T15:09:45.916+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-09T15:09:45.917+0000] {baseoperator.py:421} WARNING - ExternalTaskSensor.execute cannot be called outside TaskInstance!
[2025-02-09T15:09:45.918+0000] {external_task.py:258} INFO - Poking for tasks ['next_task'] in dag load_csv_to_postgres on 2025-02-09T15:09:40.814819+00:00 ... 
[2025-02-09T15:09:45.919+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/session.py:97: DeprecationWarning: This method is deprecated and will be removed in future.
  return func(*args, session=session, **kwargs)

[2025-02-09T15:10:00.842+0000] {local_task_job_runner.py:346} WARNING - State of this instance has been externally set to removed. Terminating instance.
[2025-02-09T15:10:00.844+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-09T15:10:00.848+0000] {process_utils.py:132} INFO - Sending 15 to group 1946. PIDs of all processes in the group: [1946]
[2025-02-09T15:10:00.850+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 1946
[2025-02-09T15:10:00.851+0000] {taskinstance.py:3093} ERROR - Received SIGTERM. Terminating subprocesses.
[2025-02-09T15:10:00.884+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-09T15:10:00.944+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=1946, status='terminated', exitcode=0, started='15:09:45') (1946) terminated with exit code 0
